packages/cli/src/networks/index.ts

```diff
@@ -77,6 +78,64 @@ export async function fetchBootnodes(network: NetworkName): Promise<string[]> {
   return enrs;
 }
 
+/**
+ * Reads and parses a list of bootnodes for a network from a file.
+ */
+export async function readBootnodes(bootnodesFilePath: string): Promise<string[]> {
```

Comment:
does this function need to be async?

Reply:
Whoops, forgot to remove the async flag :).